package.json

```diff
@@ -56,37 +53,55 @@
     "/bin",
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json",
-    "/src"
+    "/lib"
```

Comment:
Q: What do you think about adding the `yarn.lock` file here? @fvictorio  @fernandomg

Reply:
Apparently `package-lock.json` is not published, so I guess `yarn.lock` shouldn't be published either: https://stackoverflow.com/questions/44546718/should-package-lock-json-also-be-published